src/env.d.ts

```diff
@@ -1,20 +1,4 @@
-interface ShadyCSS {
-  styleElement(host: Element, overrideProps?: {[key: string]: string}): void;
-  getComputedStyleValue(element: Element, property: string): string;
-  ScopingShim: undefined|{
-    prepareAdoptedCssText(cssText: string[], name: string): void;
-  };
-  nativeShadow: boolean;
-}
-
-interface ShadyDOM {
-  inUse: boolean;
-  flush: () => void;
-}
-
 interface Window {
-  ShadyCSS?: ShadyCSS;
-  ShadyDOM?: ShadyDOM;
   ShadowRoot: typeof ShadowRoot;
```

Comment:
Why do we still need `ShadowRoot` on window? Seems like that's in `lib.dom.d.ts`...

Reply:
I was limiting the PR to only removing the parts that are now provided externally, but it builds without a lot of other stuff in here. Should I remove those also?

Reply:
(FWIW, this declaration of `ShadowRoot` is not necesary, but the part below that adds `adoptedStyleSheets` still is.)

Reply:
Actually, I think I'd rather wait to clean unrelated extra declarations out of here in another PR.

Reply:
Side note: Now that I think about it, https://github.com/lit/lit/pull/1904 was probably a mistake that will become repairable once this PR is in. Originally, I found that those types were conflicting internally with something (the original JS externs for the polyfills, I think?) so that PR avoided the problem by not adding to `Window`. Now that the polyfill types all have exactly one definition, it might be better to go back to adding Lit-specific special properties to `Window` because they _are_ actually there and a collision would indicate a real problem.

Reply:
(We talked in a meeting about this, I'm going to go ahead and merge.)